src/Rule/RemoveTrailingWhitespace.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of DOCtor-RST.
+ *
+ * (c) Oskar Stark <oskarstark@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Rule;
+
+use App\Attribute\Rule\Description;
+use App\Attribute\Rule\InvalidExample;
+use App\Attribute\Rule\ValidExample;
+use App\Value\Lines;
+use App\Value\NullViolation;
+use App\Value\RuleGroup;
+use App\Value\Violation;
+use App\Value\ViolationInterface;
+
+#[Description('Make sure there is not trailing whitespace.')]
+#[InvalidExample('Invalid sentence ')]
+#[ValidExample('Valid sentence')]
+class RemoveTrailingWhitespace extends AbstractRule implements LineContentRule
+{
+    public static function getGroups(): array
+    {
+        return [RuleGroup::Symfony()];
+    }
+
+    public function check(Lines $lines, int $number, string $filename): ViolationInterface
+    {
+        $lines->seek($number);
+        $line = $lines->current();
+
+        if ($line->raw()->match('/.+ $/')) {
```

Comment:
Is this rule expected to also detect trailing tabs?

Reply:
cc @alamirault can you add a test case for this?

Reply:
It doesn't detect tabs, I'm not sure there is this mistake in docs, because tabs are convert to space in most of IDE